SPECS/perftest/perftest.spec

```diff
@@ -36,8 +35,12 @@ chmod -x runme
 %doc README runme
 %license COPYING
 %_bindir/*
+%_mandir/man1/*.1*
```

Comment:
Should this be marked %doc as well?

Reply:
I think is only related to documentation. From what I found in Azure Linux repo, %doc is not used for manfiles, they are placed in the %files section only. 
I will try to find any official documentation on this though.